media-libs/babl/babl-0.1.74.ebuild

```diff
@@ -19,7 +19,7 @@ HOMEPAGE="http://www.gegl.org/babl/"
 
 LICENSE="LGPL-3"
 SLOT="0"
-IUSE="cpu_flags_x86_avx2 cpu_flags_x86_sse cpu_flags_x86_sse2 cpu_flags_x86_sse3 cpu_flags_x86_sse4_1 cpu_flags_x86_mmx cpu_flags_x86_f16c introspection lcms"
+IUSE="introspection lcms cpu_flags_x86_avx2 cpu_flags_x86_f16c cpu_flags_x86_mmx cpu_flags_x86_sse cpu_flags_x86_sse2 cpu_flags_x86_sse3 cpu_flags_x86_sse4_1"
```

Comment:
Sorry. I added myself here for media-libs/babl as maintainer and because @Polynomial-C already bump version before it I just made simple changes before next new release.

`virtual/pkgconfig` is moved now to `BDEPEND` in both `babl-{0.1.74,9999}.ebuild`